packages/portfolio-deploy/src/portfolio.build.js

```diff
@@ -32,7 +31,7 @@ const defaultProposalBuilder = async ({ publishRef, install }, config) => {
     getManifestCall: [
       'getManifestForPortfolio', // TODO: unit test agreemnt with getManifestForPortfolio.name
       {
-        options: { config: toExternalConfig(config, {}) },
+        options: toExternalConfig(config, {}, portfolioDeployConfigShape),
```

Comment:
At your discretion, I'd suggest namespacing `options`:

```suggestion
        options: { portfolio: toExternalConfig(config, {}, portfolioDeployConfigShape) },
```

If you would do this, I'll document this somewhere, like in the Wiki, or the `deploy-script-support/README.md`.

Reply:
This deployment stuff is scope-creep for this PR. Ideally it would have gone in separately.

I'd rather not add more manual testing to my day.

IOU an issue or something; #11456 is sort of close.
cc @LuqiPan